### ISSUE TITLE
Training is interrupted by itself

### ISSUE THREAD

**Issue by popocry** [fizyr/keras-retinanet#1545]
I am using my own training set for training. I set 100 epoch but it stops by itself at 7 epoch, and no errors are generated.
I train for many times and it will be interrupted by about 7-8epoch.

This is the code i run:
python bin/train.py --weights snapshots/resnet50_coco_best_v2.1.0.h5 --compute-val-loss --weighted-average --image-min-side 480 --image-max-side 800 --batch-size 5 --steps 1172 --epochs 100 csv CSV/train.csv CSV/class.csv --val-annotations CSV/val.csv

**Comment by nikhilcms** [comment]
I am also using my own training dataset , set 50 epoch , but it stopped at 6 itself without any warning/error.
please let us know how can we fix this issue.

**Comment by ewaty** [comment]
Same for me, it's not 7 (16 I think), but it trains for a bit and then crashes without any error. It's not a problem when I turn off validation
